tests/CheckCommandTest.php

```diff
@@ -25,6 +25,8 @@ class CheckCommandTest extends TestCase
 
     protected function setUp(): void
     {
+        parent::setUp();
```

Comment:
Good practice to call `parent::setup()` in order to avoid overriding completely the function ;)